### PR TITLE
Initial structure of the login page

### PR DESCRIPTION
**Pull request by keiffer01**
### What is a quick description of the change?

Structures index.html to to a welcome page with a sign-in button.

### Is this fixing an issue?

none

### Are there more details that are relevant?

The sign-in button will be implemented via Firebase Authentication using FirebaseUI, hence its id "firebaseui-login".

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [  ] Tests passing
- [ x ] Coding style (indentation, etc)

No testing needed, a devserver was used to ensure that elements were being displayed properly.

### Any additional comments?

none

### CODE REVIEW

File: slurp/src/main/webapp/index.html [hunk 1]
```diff
@@ -17,6 +19,19 @@
...
   </head>
+
   <body>
+
+    <div id="login-card">
+      <div class="login-title">
+        <h1 id="welcome">Welcome to</h1>
+        <h1 id="slurp">SLURP</h1>
+      </div>
+      <!-- TODO: Put path to logo when we have one. -->
```
Comment (zghera) [inline slurp/src/main/webapp/index.html:30]: May want to create an issue for this. I am not sure if this is too small for creating an issue but I think that is best practice to create an issue for any todo.

File: slurp/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,8 +1,10 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
-    <title>Slurp</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (zghera) [inline slurp/src/main/webapp/index.html:6]: Woah, I have never seen this tag before. Pretty cool stuff.

File: slurp/src/main/webapp/index.html (hunk 1 above)
Comment (beiebi0) [reply slurp/src/main/webapp/index.html:30]: +1

Comment (Hiram-Silvey) [reply slurp/src/main/webapp/index.html:30]: +1
Please open an issue for this and reference the issue in the TODO.

Comment (keiffer01) [reply slurp/src/main/webapp/index.html]: Done, good thing to keep in mind for the future too.

File: slurp/src/main/webapp/index.html (hunk 2 above)
Comment (keiffer01) [reply slurp/src/main/webapp/index.html:6]: It's a good thing to have for mobile devices and other small screens! Won't take care of everything but is usually a good first step.

**Pull request merged**